_maps/configs/minutemen_atlas.json

```diff
@@ -0,0 +1,49 @@
+	{
```

Comment:
Add `"faction": "/datum/faction/clip",` somewhere, or it's gonna show as indie on the spawning menu.